### DISCUSSION TITLE
Had anyone had any luck getting autocompletion to work on wsl2?

### DISCUSSION THREAD

**Discussion by K2ouMais** [JanDeDobbeleer/oh-my-posh#4010] [General]
Hello everyone,

when I try the command "oh-my-posh completion bash" all I get is a bunch of lines and autocompletion doesnt seem to work.

Do anybody know what I am doing wrong?

![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/52510455/187590ab-0c32-403b-a39c-f52976db3d92)

**Comment by JanDeDobbeleer** [comment]
Run oh-my-posh completion --help to see how to enable them for your shell.

**Reply by K2ouMais** [reply]
Hello there,

to get some help I had to use the ```oh-my-posh completion bash --help``` command.

After that I saw that it has to write to a file...

```oh-my-posh completion bash > /etc/bash_completion.d/oh-my-posh```

The file couldnt be created, even as ```sudo```, but I could create the file manually and had to give it all rights ```chmod 777 oh-my-posh```.

Even after the command ```source <(oh-my-posh completion bash)``` I am not getting any completion.

Maybe this doesnt work in WSL2? That could be the reason...

Thanks

**Reply by JanDeDobbeleer** [reply]
@K2ouMais this functionality is something that comes with Cobra, we don't officially support it. I'm curious why you'd want that though?

**Comment by K2ouMais** [comment]
I just found out, that bash doesnt have autosuggestions like zsh.

Sorry for the inconvenience.
